workflows/create_summary_report.nf

```diff
@@ -0,0 +1,35 @@
+include { summary_report; summary_report_fasta; summary_report_default } from './process/summary_report'
```

Comment:
Can we include this in the existing Ncov workflows?

Reply:
I think I found this on another branch, it worked for me

Reply:
Yea it works fine. I'm asking if we could include the generation of summary reports within the Ncov workflow; instead of having a seperate workflow completely to create summary report. Maybe I am misunderstanding. If so we can discuss over Slack